bftengine/src/bftengine/ViewChangeSafetyLogic.cpp

```diff
@@ -1,11 +1,14 @@
-//Concord
+// Concord
 //
-//Copyright (c) 2018 VMware, Inc. All Rights Reserved.
+// Copyright (c) 2018 VMware, Inc. All Rights Reserved.
 //
-//This product is licensed to you under the Apache 2.0 license (the "License").  You may not use this product except in compliance with the Apache 2.0 License. 
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0 License.
```

Comment:
Tagging this instance, though it occurs in many/most of these files. When I run `format-code.sh` locally, this is re-wrapped, because it's 82 characters wide here:

```
-// You may not use this product except in compliance with the Apache 2.0 License.
+// You may not use this product except in compliance with the Apache 2.0
+// License.
```

Reply:
This is a quirk in how clang-format treats very long comments. For lines where the original sentence (82 chars) was on one line it wrapped it. For one very long line it wrapped it but left that lest part at 82 chars. Re-running it fixed those remaining lines. Re-running a third time produced no changes, so I think we are good now.

I rebased and pushed up the changes.